tests/queries/0_stateless/01664_array_slice_ubsan.sql

```diff
@@ -0,0 +1 @@
+SELECT arraySlice(groupArray(x), -9223372036854775808, NULL) AS y FROM (SELECT '6553.5', uniqState(NULL) AS x FROM numbers(3) GROUP BY number);
```

Comment:
There is also UB for `select arraySlice([],  -9223372036854775808)` (in `sliceDynamicOffsetUnbounded`)